src/main/java/org/craftercms/studio/api/v2/dal/ClusterDAO.java

```diff
@@ -65,4 +65,11 @@
      * @return Cluster member with given id
      */
     ClusterMember getMemberById(long clusterMemberId);
+
+    /**
+     * Check if cluster member exists with given url
+     * @param memberUrl Member Url
+     * @return number of cluster members
+     */
+    int existsMember(String memberUrl);
```

Comment:
Rename to `getMemberIndex` or `memberExists` depending on what's being returned. See the previous question.